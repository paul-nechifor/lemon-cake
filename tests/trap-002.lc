(= f (~ (a b) ($trap (last
  (= ret 0)
  (map (range a b) (~ (a) (if (- a 10)
    (= ret (+ ret a))
    ($entrap (list 'will not go above 10' ret a))
  )))
  ret
))))

(repr (f 2 20))
(repr (f 2 5))
