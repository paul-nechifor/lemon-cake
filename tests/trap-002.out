('will not go above 10' 44 10)9
